### PR TITLE
Stop inserting search history to the DB in the search API

### PR DESCRIPTION
**Pull request by sywhb**

### CODE REVIEW

**Comment by jacksonh** [comment]
Will search history API still work?

**Comment by sywhb** [comment]
> Will search history API still work?

No, it won't work but I don't think we are calling the search history API in the client now

**Comment by jacksonh** [comment]
Yeah I suppose its fine for the clients to cache this locally for search history. It means it isn't synced across devices, but I don't know if that is a big concern

**Review by jacksonh — APPROVED** [review]

**Comment by sywhb** [comment]
I agree with you @jacksonh

The local search history works great so far

**Comment by sywhb** [comment]
@jacksonh I set a limit on the search query size to 255 characters. Is that enough?

File: packages/api/src/schema.ts [hunk 1]
```diff
@@ -2542,7 +2542,7 @@ const schema = gql`
     search(
       after: String
       first: Int
-      query: String
+      query: String @sanitize(maxLength: 255)
```
Comment (sywhb) [inline packages/api/src/schema.ts:2545]: In case you missed the change here @jacksonh

**Pull request merged**